manifests/s/Seelen/SeelenUI/2.1.3.0/Seelen.SeelenUI.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created with komac v2.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Seelen.SeelenUI
+PackageVersion: 2.1.3.0
+PackageLocale: en-US
+Publisher: Seelen
+PublisherUrl: https://github.com/eythaann
+PublisherSupportUrl: https://github.com/eythaann/Seelen-UI/issues
+PackageName: Seelen UI
+PackageUrl: https://github.com/eythaann/Seelen-UI
+License: PolyForm Strict License 1.0.0
+LicenseUrl: https://github.com/eythaann/Seelen-UI/blob/HEAD/LICENSE
+ShortDescription: Fully Customizable Desktop Environment for Windows.
+Tags:
+- desktop-application
+- dock
+- finder
+- taskbar
+- tauri
+- tauri-app
+- tiling-window-manager
+- toolbar
+- tools
+- topbar
+- web
+- web-technologies
+- windows
+- windows-11
```

```suggestion
- windows-11
ReleaseNotes: |-
  ### fix
  
  - bad user infomation on user module.
  
  Full Changelog: https://github.com/eythaann/Seelen-UI/compare/v2.1.2...v2.1.3
ReleaseNotesUrl: https://github.com/eythaann/Seelen-UI/releases/tag/v2.1.3
```

In order to keep manifests consistency.